components/ui/alert/alert.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+
+import {
+  accentBorderColors,
+  alertColors,
+  AlertProps,
+  renderAlertIcon,
+} from "./utils";
+
+import { XIcon } from "@heroicons/react/outline";
+import clsx from "clsx";
+import { CSSTransition } from "react-transition-group";
+
+export const Alert = React.forwardRef<HTMLDivElement, AlertProps>(
+  (
+    {
+      children,
+      className,
+      color = "yellow",
+      dismissible,
+      accentBorder = false,
+      icon,
+      visible = true,
+      onDismiss,
+      onDismissed,
+      ...rest
+    },
+    ref,
+  ) => {
+    const [_visible, setVisible] = React.useState(visible);
+
+    const _className = clsx(
+      alertColors(color),
+      accentBorderColors(accentBorder, color),
+      "rounded-md p-4",
```

Comment:
```suggestion
      "p-4",
```

I think we can live without `border-radius` from now on. The upcoming designs for Web 2.0 will be without rounded corners too, so let's follow the same design.